nix/pkgs/rusty_jwt_tools_ffi/default.nix

```diff
@@ -5,17 +5,28 @@
 , perl
 , gitMinimal
 }:
-
 let
-  version = "0.3.4";
+  sources = import ../../sources.nix;
+in
+with import sources.nixpkgs
+{
+  overlays = [
+    (import (fetchTarball "https://github.com/oxalica/rust-overlay/archive/master.tar.gz"))
```

Comment:
This is not a stable pin, but will change - and we don't really need a rust overlay either.

I'd propose adding a different (known-working) nixpkgs pin to `nix/sources.json` instead, and use `cargo` and `rustc` from there instead.